code/RedirectedURL.php

```diff
@@ -80,8 +80,8 @@ public function setFromQuerystring($val) {
 	}
 	
 	public function setTo($val) {
-		$val = rtrim($val,'?');
-		if($val != '/') $val = rtrim($val,'/');
+		//$val = rtrim($val,'?');
+		//if($val != '/') $val = rtrim($val,'/');
```

Comment:
This has a benefit of normalising a URL like `https://google.com////` to `https://google.com` - if you really want a trailing slash here then you should add it on again afterwards.

`if($val != '/') $val = rtrim($val,'/') . '/';`